### PR TITLE
runner no longer accepts 'tgt' and 'expr_form' arguments

### PR DESCRIPTION
**Pull request by sokratisg**
Solves #91 

Haven't dealt with the tests because I am quite rookie so please suggest if anything is pending on my side.
Compiles cleanly with
```
mvn clean install -DskipTests
```

### CODE REVIEW

**Comment by mchugh19** [comment]
That's awesome, thanks! I'll polish this up with the tests and get out an update.

File: src/main/java/com/waytta/Utils.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ public static boolean validateFunctionCall(JSONArray returnArray) {
 
         // Salt's /hook url returns non standard response. Assume this response
         // is valid
-        JSONArray successHook = JSONArray.fromObject("[{\"Success\": True}]");
```
Comment (mchugh19) [inline src/main/java/com/waytta/Utils.java]: What is this change for? It seems unrelated to the target and target_type keys.

**Comment by mchugh19** [comment]
Just handled this slightly differently in #93.
This PR had also changed one of the validation checks. Was there a reason for that? Is there a test case that is currently missing?

I'll close this out as the tgt and expr_form case is now handled and will be in the next release.

**Comment by sokratisg** [comment]
I did a test run and couldn't find ```"[{\"Success\": True}]"``` in the minion return. This is why I also changed this part.

You're right, it should have been posted in a different PR.

**Comment by mchugh19** [comment]
Cool cool. What sent that return? I can add it to the test cases, but would like to know what to call it. 

Thanks!

**Comment by sokratisg** [comment]
According to my understanding, it returns the status of the execution module.
```"return": true``` should mean it executed successfully.

**Comment by mchugh19** [comment]
Running what module causes the salt-api to respond with`[{"return": true}]`? I'd like to make sure we capture the actual output and the name of the module to help track this issue. Thanks!

**Comment by sokratisg** [comment]
Yes, you can see for yourself if you save the json output and inspect it.

Sent from a handheld device.

On 29 May 2017, at 17:37, Christian McHugh <notifications@github.com<mailto:notifications@github.com>> wrote:


Running what module causes the salt-api to respond with[{"return": true}]? I'd like to make sure we capture the actual output and the name of the module to help track this issue. Thanks!

**Comment by mchugh19** [comment]
Running what responded with `[{"return": true}]`? What was the jenkins telling the salt-api to run that jenkins got that response back?

I'd like to get the exact command to run that gives that response so that I ensure we validate the exact response. If that were a python return, I would expect it to read `True` rather than `true` for example. Thank you

**Comment by sokratisg** [comment]
Check the related issue containing the salt-bus command of the orchestrate runner

Sent from a handheld device.

On 29 May 2017, at 17:45, Christian McHugh <notifications@github.com<mailto:notifications@github.com>> wrote:


Running what responded with [{"return": true}]? What was the jenkins telling the salt-api to run that jenkins got that response back?

I'd like to get the exact command to run that gives that response so that I ensure we validate the exact response. If that were a python return, I would expect it to read True rather than true for example. Thank you
